pandas/core/resample.py

```diff
@@ -1141,6 +1141,16 @@ def _get_time_bins(self, ax):
                                         tz=tz,
                                         name=ax.name)
 
+        # GH 15549
+        # In edge case of tz-aware resapmling binner last index can be
+        # less than the last variable in data object.
+        # This leads to `Values falls after last bin` error
```

Comment:
this last sentence is not relevant; you are fixing this!
add why the bonner is too short (eg because of DST)